source/install/cloud-changelog.md

```diff
@@ -3,12 +3,45 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-11-10](#release-2021-11-10)
 - [Release 2021-10-27](#release-2021-10-27)
 - [Release 2021-10-13](#release-2021-10-13)
 - [Release 2021-09-29](#release-2021-09-29)
 - [Release 2021-08-12](#release-2021-08-12)
 - [Release 2021-07-29](#release-2021-07-29)
-- [Release 2021-07-15](#release-2021-07-15)
+
+## Release 2021-11-10
+
+### Improvements
+
+#### User Interface (UI)
+ - Added online status to profile images on user autocomplete.
+ - App Commands now have an option to open them as modals.
+ - Added support for navigating through Collapsed Reply Threads via arrow keys.
+ - Added support for focusing the input box in Collapsed Reply Threads while typing.
+ - Added support for blurring the input box in Collapsed Reply Threads on pressing Escape.
+ - Adjusted the channel override desktop notification preference for Threads.
+ - Added a **Click to open thread** setting for all users.
+ - Added lang/tz defaults to Luxon.
+ - User interface is now improved when no text is set for a custom status.
+
+#### Administration
+ - Channel results in channel switcher will include channels from other teams as well as private channels. For users using Bleve or Elasticsearch for autocomplete/search would have to reindex their data to get the new results. Since this can take a long time, it is suggested to disable autocomplete and run indexing in the background. And then when it's complete, re-enable autocomplete.
```

Comment:
I don't think this applies to Cloud

Reply:
To confirm, we don't use Elasticsearch in cloud?

Reply:
@agnivade correct, no Elasticsearch in cloud at the mometn

Reply:
Removing,